documentation/modules/oauth/proc-oauth-broker-config.adoc

```diff
@@ -7,19 +7,22 @@
 
 This procedure describes how to configure Kafka brokers so that the broker listeners are enabled to use {oauth} authentication using an authorization server.
 
+{oauth} requires configuration of TLS listeners.
+Plain listeners are not recommended.
+
 If the authorization server is using certificates signed by the trusted CA and matching the {oauth} server hostname, TLS connection works using the default settings.
-Otherwise, you have two connection options for your external listener configuration when delegating token validation to the authorization server:
+Otherwise, you have two connection options for your listener configuration when delegating token validation to the authorization server:
 
 . TLS connection to the authorization server with trusted certificates
 . Direct connection using an introspection endpoint configuration
 
-Both options are described in this procedure.
-
+Examples of the connection options are described in this procedure.
```

Comment:
Can we add a link or copy paste the example here?

Reply:
I've made the preceding bullets links instead and removed this sentence.